_harp/_data.json

```diff
@@ -1,51 +1,49 @@
-[  
-   {  
+[
+   {
       "name":"Jack Frysinger",
       "twitter":"jackfrys",
-      "www":"jackfrysinger.com",
-      "desc":"",
+      "www":"//jackfrysinger.com",
       "github":"jackfrys",
-      "interests":[  
+      "interests":[
          "Server programming",
          "Apple fanboying",
          "Voice-driven interactions",
          "iOS app development",
          "Dunkin' Donuts"
       ],
-      "langs":[  
+      "langs":[
          "Java",
          "Python",
          "Swift",
          "LaTeX"
       ]
    },
-   {  
+   {
       "name":"Danielle Nguyen",
-      "twitter":"My tumblr is http://smurtleturtle.tumblr.com/ bc I'm cultured and refined",
```

Comment:
@daniellenguyen this whole string was getting piped into an `<a>` tag. Feel free to fix, or add a Tumblr attribute to the template.

Reply:
yes i was being a lazy butt. will do